webapp/packages/core-blocks/src/Snackbars/ProcessSnackbar.tsx

```diff
@@ -54,12 +54,13 @@ export const ProcessSnackbar: NotificationComponent<ProcessSnackbarProps> = obse
     return null;
   }
 
+  function close() {
+    onCancel?.();
+    notification.close(false);
+  }
```

Comment:
notification can be closed programmatically, because of that, you need to listen for notification close event and invoke the close handler